src-js/experiments/relatedResources.js

```diff
@@ -0,0 +1,19 @@
+$(() => {
+  // https://app.optimizely.com/v2/projects/16812830475/experiments/20638820544/variations
+  window.OptimizelyClient.getVariationName({
+    experimentKey: 'dd_docs_related_resources_mvp_test',
+    groupExperimentName: 'q3_fy22_docs_disco_experiment_group_test'
+  }).then(variation => {
+    if (variation === "treatment") {
+      $('#related-resources').show();
+      // track clicks on each items
+      $("#related-resources li a").click(function () {
+        window.AnalyticsClient.trackAction('related-rerouces-clicked', {
```

Comment:
small typo here to look out for:
```suggestion
        window.AnalyticsClient.trackAction('related-resources-clicked', {
```